package.json

```diff
@@ -43,7 +42,7 @@
     "bs-jest": "^0.3.2",
     "eslint-plugin-react-hooks": "^4.3.0",
     "postcss": "^8.4.5",
-    "rescript": "^9.1.4",
+    "rescript": "^11.0.0-beta.1",
```

Comment:
Don't get scared by the beta version. This is almost the golden RC and we test it heavily with other production codebases.